recipe for target 'cloudcore' failed

Hello Team

Please Guide us to when i execute command "make all WHAT=cloudcore" get the error.
 

root@RPCHW002:/src/github.com/kubeedge/kubeedge# make all WHAT=cloudcore
cd cloud && make cloudcore
make[1]: Entering directory '/src/github.com/kubeedge/kubeedge/cloud'
bash ../hack/verify-golang.sh
go detail version: go version go1.13.4 linux/amd64
go version: 1.13.4
go build -ldflags '-X github.com/kubeedge/kubeedge/pkg/version.buildDate=2019-11-27T09:18:41Z -X github.com/kubeedge/kubeedge/pkg/version.gitCommit=e9bbff005d607cb2ef84b9496ba977dac37dca99 -X github.com/kubeedge/kubeedge/pkg/version.gitTreeState=clean -X github.com/kubeedge/kubeedge/pkg/version.gitVersion=v1.1.0-beta.0.247+e9bbff005d607c -X github.com/kubeedge/kubeedge/pkg/version.gitMajor=1 -X github.com/kubeedge/kubeedge/pkg/version.gitMinor=1+' cmd/cloudcore/cloudcore.go
go: k8s.io/apimachinery@v0.0.0-20190612165923-1799e75a0719: invalid pseudo-version: does not match version-control timestamp (2019-06-12T20:58:21Z)
Makefile:11: recipe for target 'cloudcore' failed
make[1]: *** [cloudcore] Error 1
make[1]: Leaving directory '/src/github.com/kubeedge/kubeedge/cloud'
Makefile:12: recipe for target 'all' failed
make: *** [all] Error 2



Thanks In Advance.

--- a/hack/verify-golang.sh
+++ b/hack/verify-golang.sh
@@ -37,7 +37,7 @@ if [ $X -lt 1 ] ; then
 	exit 1
 fi
 
-if [ $Y -lt 12 ] ; then  
-	echo "go minor version must >= 12, now is $Y"
+if [ $Y -ne 12 ] ; then  
+	echo "go minor version must == 12, now is $Y"
 	exit 1
 fi

